platform/linux-generic/include/odp/api/plat/packet_inlines.h

```diff
@@ -21,6 +21,9 @@
 /** @internal Inline function offsets */
 extern const _odp_packet_inline_offset_t _odp_packet_inline;
 
+/** @internal Pool inline function offsets */
+extern const _odp_pool_inline_offset_t _odp_pool_inline;
```

Comment:
extern is not needed in .h file, also extern 2 lines above.

Reply:
In this case it is needed. _odp_pool_inline is defined in odp_pool.c and _odp_packet_inline in odp_packet.c. ODP build with '--disable-abi-compat' will fail without extern declarations.

Reply:
ok, then.

Reply:
@muvarov extern is not needed for function prototypes.